qemu/tests/block_stream_check_backingfile.py

```diff
@@ -36,6 +36,13 @@ def check_backingfile(self):
                    "no backing file")
             raise error.TestFail(msg)
 
+    def check_backingfile_exist(self):
+        backingfile = self.get_backingfile()
```

Comment:
@vivianQizhu  if absolute_path = no, seems this backingfile will always not equal to self.base_image. Why not use self.backingfile?